lib/utils/speech-data/tts-cartesia.js

```diff
@@ -0,0 +1,301 @@
+/* eslint-disable max-len */
+module.exports = [
```

Comment:
are these only used if the API to list voices failed?  Because we can get these from the api riught?